### PR TITLE
Voice calls feature

### PR DESCRIPTION
**Pull request by goncalvesjoao**
please mastah @joaquimadraz could you please be so kind as to review this humble PR on @ricardomarquesramos 's behalf.

Much obliged.

### CODE REVIEW

File: spec/sms_broker/twilio_voice_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        SmsBroker.clear_setup
+      end
+
+      def send_message(text_message, to)
+        SmsBroker.service.message(text_message).to(to).deliver
+      end
+
+      context 'valid' do
+        it 'should send message with success' do
+          # binding.pry
```
Comment (joaquimadraz) [inline spec/sms_broker/twilio_voice_spec.rb]: @goncalvesjoao missing some expectations here

File: spec/sms_broker/real_call_spec.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 describe SmsBroker do
   context 'SmsBroker' do
     let(:text_message) { 'Hello World' }
+    let(:voice_text_message) { 'Alex,, please give me a raise,, please!' }
```
Comment (joaquimadraz) [inline spec/sms_broker/real_call_spec.rb]: @goncalvesjoao I think we find a more suitable message 🌮

File: spec/sms_broker/nexmo_voice_spec.rb [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          stub_nexmo_create_voice_message_unknown_error \
+            from_phone, to_phone, voice_message
+
+          response = SmsBroker.voice_message(voice_message).to(to_phone).deliver
+
+          expect(response.success?).to eq(false)
+          expect(response.serialized.length).to be > 0
+        end
+
+        xit 'should return error an invalid language' do
```
Comment (joaquimadraz) [inline spec/sms_broker/nexmo_voice_spec.rb]: @goncalvesjoao is this test still valid?

Comment (goncalvesjoao) [inline spec/sms_broker/nexmo_voice_spec.rb]: @ricardomarquesramos ?

File: spec/sms_broker/real_call_spec.rb (hunk 2 above)
Comment (goncalvesjoao) [inline spec/sms_broker/real_call_spec.rb]: @ricardomarquesramos ?

File: spec/sms_broker/twilio_voice_spec.rb (hunk 1 above)
Comment (goncalvesjoao) [inline spec/sms_broker/twilio_voice_spec.rb]: @ricardomarquesramos ?

Comment (ricardomarquesramos) [inline spec/sms_broker/twilio_voice_spec.rb]: Twilio is not implemented so

File: spec/sms_broker/nexmo_voice_spec.rb (hunk 3 above)
Comment (ricardomarquesramos) [inline spec/sms_broker/nexmo_voice_spec.rb]: I was starting to implement the param 'language' send to the voice message, but i actually wasn't able to finish the test case of passing an invalid language param

**Comment by AlexMC** [comment]
@ricardomarquesramos I need to push this to staging asap, can you tidy things up according to the feedback?

**Comment by ricardomarquesramos** [comment]
Tied up according to feedback

**Comment by AlexMC** [comment]
@joaquimadraz happy with the changes?

File: lib/sms_broker/client/twilio.rb [hunk 4]
```diff
@@ -24,6 +24,11 @@ def send_message(message)
         Response::TwilioError.new(exception)
       end
 
+      def send_voice_message(message)
+        # Not implemented yet
+        raise SmsBroker::Exceptions::InvalidService
```
Comment (joaquimadraz) [inline lib/sms_broker/client/twilio.rb]: @ricardomarquesramos 
This exception is to be used in another context, when there's an invalid configuration. 
Better create an `Exception::NotImplemented`.

**Pull request merged**